db/users.go

```diff
@@ -211,9 +208,7 @@ func (dbc *DatabaseContext) UpdatePrincipal(ctx context.Context, updates *auth.P
 			if updates.JWTChannels != nil && updatedJWTChannels.UpdateAtSequence(updates.JWTChannels, nextSeq) {
 				user.SetJWTChannels(updatedJWTChannels, nextSeq)
 			}
-			if updates.JWTLastUpdated != nil {
-				user.SetJWTLastUpdated(*updates.JWTLastUpdated)
-			}
+			user.SetJWTLastUpdated(time.Now())
```

Comment:
I don't think we want to set JWTLastUpdated if no other JWT properties have changed, do we?  i.e. we don't want to be updating this property if admin channels is the only thing being changed.

Reply:
That's fair - I'll add a 2nd bool to track whether changes were the result of JWT claims.